src/api/solver/content-types/solver/schema.json

```diff
@@ -0,0 +1,64 @@
+{
+  "kind": "collectionType",
+  "collectionName": "solvers",
+  "info": {
+    "singularName": "solver",
+    "pluralName": "solvers",
+    "displayName": "Solver",
+    "description": ""
+  },
+  "options": {
+    "draftAndPublish": false
+  },
+  "pluginOptions": {},
+  "attributes": {
+    "name": {
+      "type": "string",
+      "required": true
+    },
+    "address": {
+      "type": "string",
+      "regex": "^(0x)?[0-9a-fA-F]{40}$",
+      "required": true
+    },
+    "payoutAddress": {
+      "type": "string",
+      "regex": "^(0x)?[0-9a-fA-F]{40}$"
+    },
+    "networks": {
+      "type": "relation",
+      "relation": "manyToMany",
+      "target": "api::network.network",
+      "inversedBy": "solvers"
+    },
+    "active": {
+      "type": "boolean",
+      "default": true,
+      "required": true
+    },
+    "image": {
+      "type": "media",
+      "multiple": false,
+      "required": false,
+      "allowedTypes": [
+        "images"
+      ]
+    },
+    "organization": {
+      "type": "string"
+    },
+    "website": {
+      "type": "string",
+      "regex": "^https?:\\/\\/(www\\.)?[-a-zA-Z0-9@:%._\\+~#=]{1,256}\\.[a-zA-Z0-9()]{1,6}\\b([-a-zA-Z0-9()@:%_\\+.~#?&//=]*)"
+    },
+    "description": {
+      "type": "string"
+    },
+    "solverId": {
+      "required": true,
+      "unique": true,
+      "default": 0,
+      "type": "integer"
```

Comment:
Why an integer?

Sorry if I gave that impression.
My idea with this field is to match the ids used in the backend, which are actually strings.

For example, there's a solver named `gnosis_1inch`.

I want to use that as the id here, as for the `name` we could have `Gnosis 1inch solver` (that's why I proposed to change it to `displayName`.

Reply:
I see, I can call it `displayName` no worries. What do I call this solverId? you are happy with solverId (normally ID are UUID or numbers so might be confusing). Alternative I could call this the `name` but might also be confusing.

Reply:
I don't have anything better.
Yes, not numeric neither UUID, but still a unique identifier of the solver that's not meant to be nice for displaying.